collectibles.go

```diff
@@ -0,0 +1,26 @@
+package blockatlas
+
+type Collection struct {
+	Name string `json:"name"`
+	Symbol string `json:"symbol"`
+	ImageUrl string `json:"image_url"`
+	Description string `json:"description"`
+	External_Link string `json:"external_link"`
```

Comment:
Underscores are quite uncommon in exported Go fields, I think it should be `ExternalLink`